### ISSUE TITLE
jupyter notebook is broken on pylance v2023.10.50

### ISSUE THREAD

**Issue by ladyrick** [microsoft/pylance-release#5041]
## Environment data

-   Language Server version: v2023.10.50
-   OS and version: mac
-   Python version (& distribution if applicable, e.g. Anaconda): 3.8

## Code Snippet

```python
import plotly.express as px
fig = px.line(x=[1,2,3], y=[4,5,6], color="red", title='Life expectancy in Canada')
fig.show()
```

## Repro Steps

1. write some code in jupyter notebook.

## Expected behavior

code highlight.
<img width="779" alt="image" src="https://github.com/microsoft/pylance-release/assets/10379994/3285bfdb-635f-4a14-b592-45320b556f47">

## Actual behavior

no code highlight. roll back to 2023.10.40 fixes this bug.
<img width="834" alt="image" src="https://github.com/microsoft/pylance-release/assets/10379994/2d1bb420-ebc4-4c5f-b281-2cb3ec0204a0">

## Logs

```
Error: Debug Failure. Unable to find notebook URI for vscode-notebook-cell://xxx
Code: -32603
```

**Comment by debonte** [comment]
Dupe of https://github.com/microsoft/pylance-release/issues/5037?

**Comment by MySlientWind** [comment]
It seems that the issue can be temporarily resolved by rolling back to 2023.10.21 (pyright 088ebaa5).

**Comment by ladyrick** [comment]
> Dupe of #5037?
yes
